packages/kmod-5.15-nvidia/kmod-5.15-nvidia.spec

```diff
@@ -126,6 +126,11 @@ pushd NVIDIA-Linux-%{_cross_arch}-%{tesla_ver}/kernel-open
 # we don't set this flag the compilation fails
 make %{?_smp_mflags} ARCH=%{_cross_karch} IGNORE_CC_MISMATCH=1 SYSSRC=%{kernel_sources} CC=%{_cross_target}-gcc LD=%{_cross_target}-ld
 
+# Strip symbols out of the .ko files
+for module in *.ko; do
+  %{_cross_target}-strip -g --strip-unneeded "${module}"
+done
```

Comment:
Does it work instead to pass `INSTALL_MOD_STRIP=1` to `make`, as we do in the normal kernel builds?

Reply:
Let me give this a try.

Reply:
This didn't work. The kernel modules weren't stripped:

```
└─> ❯ ll ./x86_64-bottlerocket-linux-gnu/sys-root/usr/share/nvidia/open-gpu/drivers/
Permissions Size User     Date Modified Name
.rwxr-xr-x@ 4.8M agarrcia  9 May 17:31  nvidia-drm.ko
.rwxr-xr-x@ 3.3M agarrcia  9 May 17:31  nvidia-modeset.ko
.rwxr-xr-x@ 230k agarrcia  9 May 17:31  nvidia-peermem.ko
.rwxr-xr-x@  61M agarrcia  9 May 17:31  nvidia-uvm.ko
.rwxr-xr-x@  23M agarrcia  9 May 17:31  nvidia.ko
```